mitodl/open-edx-plugins
# feat: add basic data fields and design (CMS)

### What are the relevant tickets?
https://github.com/mitodl/hq/issues/6079

### Description (What does it do?)
- Adds the initial fields required for the chatbot
- Adds the basic form for adding all the configuration fields for the chat:
    - Prompt text
    - Enabled
    - Additional solution
    - GPT model selection (Configurable)

- Adds filtering and allows this aside only on the problem and video blocks
- Enables the user to save the data in these fields


### Screenshots (if appropriate):


https://github.com/user-attachments/assets/39a90c24-41de-49f6-9511-4e37c3b6b4e4


### How can this be tested?
- Generate the Gzip of this package using pants
- Follow the setup required configurations from https://github.com/mitodl/open-edx-plugins/pull/393
- Install that in CMS
- Add any Video or Problem block and you should see the configurations form
- Add values and save It should save successfully
- Refresh and check the values retain
- Edit any configuration in the form and save again and check that it persists


### Additional Context
